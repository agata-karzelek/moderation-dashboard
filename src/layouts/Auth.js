import React from "react";
import { CssBaseline } from "@material-ui/core";

const Auth = ({ children }) => {
  return (
    <>
      <CssBaseline />
      {children}
    </>
  );
};

export default Auth;
