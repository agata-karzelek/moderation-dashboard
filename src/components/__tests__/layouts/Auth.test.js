import React from "react";
import { shallow } from "enzyme";
import Auth from "../../../layouts/Auth";

describe("Test Case For Auth", () => {
  test("should render router", () => {
    const wrapper = shallow(<Auth />);
    expect(wrapper).toMatchSnapshot();
  });
});
