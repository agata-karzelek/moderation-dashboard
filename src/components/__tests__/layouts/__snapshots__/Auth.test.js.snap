// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Case For Auth should render router 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
</Fragment>
`;
